### PR TITLE
[DOCS] add instructions to create meson.build in how-to-create-model.md

### PR DESCRIPTION
**Pull request by EunjuYang**
[DOCS] add instructions to create `meson.build` in `how-to-create-model.md`

* This PR adds a missing part in `docs/how-to-create-model.md`.
* It includes some explanations to write `meson.build` file of a new application (under `Applications/MyApp/jni/` so as to build it.

### CODE REVIEW

**Review by skykongkong8 — APPROVED** [review]
Looks good to me!

File: docs/how-to-create-model.md [hunk 1]
```diff
@@ -125,6 +125,35 @@ int main(int argc, char *argv[]) {
...
+
+```
+myapp_sources= [
+    'main.cpp',
+    cifar_path / 'cifar_dataloader.cpp'
+]
+
+myapp_dependencies = [app_utils_dep,
+  nntrainer_dep,
+  nntrainer_ccapi_dep
```
Comment (myungjoo) [inline docs/how-to-create-model.md:143]: ```nntrainer_ccapi_dep``` includes ```nnstrainer_dep```. You don't need to add ```nntrainer_dep``` here: https://github.com/nnstreamer/nntrainer/blob/main/api/ccapi/meson.build

File: docs/how-to-create-model.md [hunk 2]
```diff
@@ -125,6 +125,35 @@ int main(int argc, char *argv[]) {
...
+
+e = executable('nntrainer_myapp',
+  myapp_sources,
+  include_directories: [include_directories('.'), cifar_include_dir],
+  dependencies: myapp_dependencies,
+  install: get_option('install-app'),
+  install_dir: application_install_dir
+)
+```
+
```
Comment (DonghakPark) [inline docs/how-to-create-model.md:154]: How about moving this explain to below contents

```
Add the following code at the end of the "Applications/meson.build": "subdir('MyApp/jni')".
...
subdir('MyApp/jni')

```

Line 25

**Review by DonghakPark — APPROVED** [review]

File: docs/how-to-create-model.md [hunk 3]
```diff
@@ -125,6 +125,35 @@ int main(int argc, char *argv[]) {
...
+### Build and Run "MyApp" application
```
Comment (djeong20) [inline docs/how-to-create-model.md:155]: how about changing the title? (it's the same as line 127!)

**Review by jijoongmoon — APPROVED** [review]
LGTM!

File: docs/how-to-create-model.md (hunk 1 above)
Comment (EunjuYang) [reply docs/how-to-create-model.md:143]: I totally missed it. Thank you for pointing it out.

File: docs/how-to-create-model.md (hunk 2 above)
Comment (EunjuYang) [reply docs/how-to-create-model.md:154]: Absolutely! it would be more readable. Thnx.

File: docs/how-to-create-model.md (hunk 3 above)
Comment (EunjuYang) [reply docs/how-to-create-model.md:155]: Make sense. Thank you for your comment!

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Pull request merged**